src/atoms/Checkbox/Checkbox.native.js

```diff
@@ -0,0 +1,166 @@
+import React, { useState, useCallback, useContext } from 'react';
+import PropTypes from 'prop-types';
+import { Text, TouchableOpacity, View } from 'react-native';
+import styled, { ThemeContext } from 'styled-components/native';
+import CheckBoxIcon from './CheckboxIcon';
+
+const CheckboxContainer = styled.View`
+  display: flex;
+  flex-direction: row;
+`;
```

Comment:
No need to create explicit container. Compose with `<Flex />` directly.

Reply:
There is already an PR pending which has `<Flex />` component created in it. Waiting for it to get merged. Will change this once it is merged.